packages/ui/views/welcome/welcome-step-four/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+
+import { View, Text, Dimensions } from 'react-native'
+
+import style from 'theme/slide'
+
+import { largeImage } from 'dimen/image'
+
+import { welcomeRunEthereum, welcomeFirstLendingApp } from 'language'
+
+import Lndr from 'ui/components/images/lndr-text'
+import Blockchain from 'ui/components/images/blockchain'
+
+export default class WelcomeStepFourView extends Component {
+  render() {
+    return (
+      <View style={style.topView}>
+        <Lndr/>
+        <Text style={[style.text, style.topSpacing]}>{welcomeFirstLendingApp}</Text>
+        <Blockchain size={largeImage}/>
```

Comment:
This gets pretty confusing - it's not a "Blockchain" component (think about this creating an instance of Blockchain) - there's nothing here that makes it seem like an image. Maybe we can re-work them all to be the same component as in `<ThemeImage name='blockchain' size={largeImage} />` or similar.

Reply:
Still keep the unique imports but re-export them under ThemeImage perhaps.